app/pages/lab/export-workflows-dialog.jsx

```diff
@@ -27,10 +27,21 @@ class ExportWorkflowsDialog extends React.Component {
       return;
     }
 
-    apiClient.type('workflows')
-      .get(project.links.workflows)
+    const order = (project.configuration && project.configuration.workflow_order) || [];
+
+    // TODO: this API call duplicates information fetched to draw the lab sidebar.
+    // when we do the lab refactor, this should be cached somewhere
+    project.get('workflows', { fields: 'display_name' })
       .then((workflows) => {
-        this.setState({ workflows });
+        let result = workflows;
+
+        if (order) {
```

Comment:
This is actually quite tricky to accomplish since the export is started by clicking a button in a modal dialog. I'd like to merge this PR without that behavior unless you feel strongly about it.